### PR TITLE
⬆️ Update sample applications

### PR DESCRIPTION
**Pull request by ElRaptorus**
## Changes

Update all sample applications so that they reflect the current state of things.

This also gets rid of some SecurityAlerts concerning an obsolete version of Sequelize.

## Issues

PR: #82

## How to test the changes
 
Run the sample applications as described in their respective README files.

### CODE REVIEW

File: samples/02_external_process_engine/client_no_ioc_container/src/program.ts [hunk 1]
```diff
@@ -57,46 +45,44 @@ async function executeSample(): Promise<void> {
...
   // Get a list of all waiting UserTasks, using the processModelId and the correlationId.
-  const waitingUserTasks: UserTaskList =
-    await consumerApiClientService.getUserTasksForProcessModelInCorrelation(identity, processModelId, correlationId);
+  const waitingUserTasks = await consumerApiClient.getUserTasksForProcessModelInCorrelation(identity, processModelId, correlationId);
 
   // There should be one waiting UserTask.
-  const userTask: UserTask = waitingUserTasks.userTasks[0];
-  logger.info(`Found a wating UserTask with key ${userTask.id}`);
+  const userTask = waitingUserTasks.userTasks[0];
+  logger.info(`Found a wating UserTask with ID ${userTask.id}`);
```
Comment (rrrene) [inline samples/02_external_process_engine/client_no_ioc_container/src/program.ts]:
```suggestion
  logger.info(`Found a waiting UserTask with ID ${userTask.id}`);
```

File: samples/01_internal_process_engine/config/demo/consumer_api/external_accessor.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "socketUrl": "http://localhost:9999"
+}
```
Comment (alexanderkasten) [inline samples/01_internal_process_engine/config/demo/consumer_api/external_accessor.json]:
```suggestion
}

```

**Comment by ElRaptorus** [comment]
@rrrene done

File: samples/01_internal_process_engine/src/setup.ts [hunk 3]
```diff
@@ -5,35 +5,44 @@ import {InvocationContainer} from 'addict-ioc';
...
 
 import {AppBootstrapper} from '@essential-projects/bootstrapper_node';
+import {HttpExtension} from '@essential-projects/http_extension';
 import {IIdentity} from '@essential-projects/iam_contracts';
 
-import {IProcessModelService} from '@process-engine/process_engine_contracts';
+import {IProcessModelService} from '@process-engine/process_model.contracts';
 
-const logger: Logger = Logger.createLogger('ssample:internal:setup');
+const logger = Logger.createLogger('ssample:internal:setup');
```
Comment (rrrene) [inline samples/01_internal_process_engine/src/setup.ts]: Ist **s**sample hier richtig?

File: samples/02_external_process_engine/server/src/application.ts [hunk 4]
```diff
@@ -89,16 +106,17 @@ async function start(): Promise<void> {
...
   const dummyIdentity: IIdentity = {
-    token: 'defaultUser',
+    token: 'ZHVtbXlfdG9rZW4=',
+    userId: 'dummy_token',
   };
 
-  const xml: string = readProcessModelFromFile(processFileName);
+  const xml = readProcessModelFromFile(processFileName);
 
   // Get the ProcessModelService, which ahndles the import of ProcessModels.
```
Comment (rrrene) [inline samples/02_external_process_engine/server/src/application.ts]:
```suggestion
  // Get the ProcessModelService, which handles the import of ProcessModels.
```

File: samples/01_internal_process_engine/src/setup.ts (hunk 3 above)
Comment (ElRaptorus) [reply samples/01_internal_process_engine/src/setup.ts]: nope.

Comment (ElRaptorus) [reply samples/01_internal_process_engine/src/setup.ts]:
```suggestion
const logger = Logger.createLogger('sample:internal:setup');
```

**Comment by ElRaptorus** [comment]
@rrrene done

**Pull request merged**